diff -Naur ShivaVG-0.2.1/configure.in ShivaVG-0.2.1.tpg/configure.in
--- ShivaVG-0.2.1/configure.in	2009-03-20 03:27:52.000000000 +0000
+++ ShivaVG-0.2.1.tpg/configure.in	2021-10-12 19:20:43.423459236 +0000
@@ -181,7 +181,7 @@
 # JPEG library required for VGImage example
 
 CFLAGS="-I$prefix/include"
-LDFLAGS="-ljpeg -L/usr/local/lib -L$prefix/lib"
+LDFLAGS="-ljpeg"
 
 AC_CHECK_HEADERS(
 	[jpeglib.h],
diff -Naur ShivaVG-0.2.1/examples/Makefile.am ShivaVG-0.2.1.tpg/examples/Makefile.am
--- ShivaVG-0.2.1/examples/Makefile.am	2009-03-20 03:27:52.000000000 +0000
+++ ShivaVG-0.2.1.tpg/examples/Makefile.am	2021-10-12 19:21:09.923227575 +0000
@@ -9,7 +9,7 @@
 EXAMPLE_SRCS = test.h test.c
 EXAMPLE_CF = @CONFIG_CFLAGS@ -I${INCLUDE_VG_DIR} -I${prefix}/include
 EXAMPLE_LA = @CONFIG_LDADD@ ${LIB_VG}
-EXAMPLE_LF = @CONFIG_LDFLAGS@ -L${prefix}/lib
+EXAMPLE_LF = @CONFIG_LDFLAGS@
 
 EXTRA_DIST = *.jpg *.png
 
diff -Naur ShivaVG-0.2.1/examples/Makefile.in ShivaVG-0.2.1.tpg/examples/Makefile.in
--- ShivaVG-0.2.1/examples/Makefile.in	2009-03-20 03:27:52.000000000 +0000
+++ ShivaVG-0.2.1.tpg/examples/Makefile.in	2021-10-12 19:21:21.223130461 +0000
@@ -280,7 +280,7 @@
 EXAMPLE_SRCS = test.h test.c
 EXAMPLE_CF = @CONFIG_CFLAGS@ -I${INCLUDE_VG_DIR} -I${prefix}/include
 EXAMPLE_LA = @CONFIG_LDADD@ ${LIB_VG}
-EXAMPLE_LF = @CONFIG_LDFLAGS@ -L${prefix}/lib
+EXAMPLE_LF = @CONFIG_LDFLAGS@
 EXTRA_DIST = *.jpg *.png
 test_vgu_SOURCES = \
 	${EXAMPLE_SRCS} test_vgu.c
